src/main/java/core/util/ConnectionUtil.java

```diff
@@ -0,0 +1,28 @@
+package core.util;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    static {
+        try {
+            Class.forName("com.mysql.cj.jdbc.Driver");
+        } catch (ClassNotFoundException e) {
+            throw new RuntimeException("Can't find MySQL Driver", e);
+        }
+    }
```

Comment:
Optional - you can delete this block. Or you can`t get a connection without him?

Reply:
I can, but this is how Bohdan did this in the video tutorial)